KishonStClair-Fatima.js

```diff
@@ -0,0 +1,83 @@
+// Question 1
+//Write a function calculateCube that takes a single number and prints the volume of a cube made from that number.
+
+function calculateCube(number) {
+  
+    console.log(number * number * number)
+}
+calculateCube(5)
+
+
+// Question 2
+//  write a function with an parameter for a string
+function isAVowel(str) {
+    // write a agrument for the length of the string and determine its a vowel
+    let count = 0
+    const vowels = "abpurtRWKO"
+    for(var i = 0; i < str.length; i++){
+    console.log(vowels.indexOf(str[i]) > -1)
+    }
+    
+  } 
+  
+  isAVowel("a")
```

Comment:
this works, but you're only checking against the letters in "abpurtRWKO". This means that if I do `isAVowel('e')`, I get `false`, but if I do `isAVowel('R')`, I get `true`. I think it would make sense to change the variable `vowels` to be `const vowels = 'aeiouAEIOU'`. This way, you account for all vowels as lowercase or capitals, and you don't have any consonants in there.